lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1030,7 +1030,7 @@ def get_proj(self):
 
         self.eye = E
         self.vvec = R - E
-        self.vvec = self.vvec / proj3d.mod(self.vvec)
+        self.vvec = self.vvec / proj3d._mod(self.vvec)
```

Comment:
This is going to conflict with #13020.

Reply:
I'll rebase #13030 on top of this after this is merged.